doc/source/whatsnew/v1.2.4.rst

```diff
@@ -29,7 +29,7 @@ Fixed regressions
 Bug fixes
 ~~~~~~~~~
 
--
+- Bug in groupby aggregation when using an object-typed index (issue:`40014`)
```

Comment:
we are rarely going to target anything but master 
move to 1.3

Reply:
Good to know. Updated